src/resolve/resolved-dns-cache.c

```diff
@@ -18,6 +18,9 @@
 /* We never keep any item longer than 2h in our cache */
 #define CACHE_TTL_MAX_USEC (2 * USEC_PER_HOUR)
 
+/* The max clamp tll for stale data is set to 30seconds. */
+#define CACHE_STALE_CLAMP_TTL_MAX_USEC (30 * USEC_PER_SEC)
```

Comment:
why so short?

Reply:
or hmm, let me rephrase this: why so long?

i think we should probably set this to 1s, because we are already a local cache, and we'd rather have clients ask us again then to cache our response... given that talking to us, a local service is cheap effectively disabling caching in clients further up the pipeline should not be an issue